en/collect/import-using-online-bibliographic-database.md

```diff
@@ -173,7 +173,36 @@ To be detailed.
 
 [Springer](https://www.springer.com/) \(aka Springer Science+Business Media\) is a global publishing company that publishes books, e-books, and peer-reviewed journals in science, technical and medical publishing. Springer also hosts a number of scientific databases, including SpringerLink, Springer Protocols, and SpringerImages \([Wikipedia](https://en.wikipedia.org/wiki/Springer_Science%2BBusiness_Media)\).
 
-### zbMATH
-
-[zbMATH](https://www.zbmath.org/) is a reviewing service providing reviews and abstracts for articles in pure and applied mathematics \([Wikipedia](https://en.wikipedia.org/wiki/Zentralblatt_MATH)\).
-
+### zbMATH Open
+
+[zbMATH Open](https://zbmath.org/) is an abstracting and reviewing service in pure and applied mathematics. Its database contains about 4 million bibliographic entries with reviews or abstracts currently drawn from about 3,000 journals and book series, and 180,000 books. The coverage starts in the 18th century and is complete from 1868 to the present by the integration of the "Jahrbuch über die Fortschritte der Mathematik" database \([about](https://zbmath.org/about/)\).
+
+#### Structured Search
+
+You can use the same query syntax as in the one-line search at zbmath.org. This means that your query can be composed of several terms, combined by the logical operators `&` (and) and `|` (or). Queries are case-insensitive. Further operators that can be used are `!` for logical negation, `*` for a right wildcard, `“ “` for exact phrase matches, and parentheses `( )` to group terms. Optionally, it is possible to add a field name in the form field:text to limit the search results. The supported fields are:
+
+* **any**: anywhere (default)
+* **au**: author, editor
+* **ti**: title
+* **so**: source
+* **py**: publication year
+* **cc**: MSC code
+* **dt**: document type (possible values are **j** for journal articles, **b** for books, **a** for book articles)
+* **an**: the zbl id of the document
+* **ai**: internal author identifier
+* **la**: language either as a string or as [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) language code
+* **ab**: search for term in reviews or abstracts
+* **rv**: reviewer
+
+#### Examples 
```

Comment:
These examples are directly taken from the help page of zbmath.org. I believe we are not allowed to do so (because of copyright). Please, provide new examples or link to the zbmath help page.

Reply:
I deleted the examples that where teken from the zbmath help page and added some new ones.